index.html

```diff
@@ -1085,6 +924,213 @@ <h3>Implicit Release of Pointer Capture</h3>
 
         </section>
     </section>
+    <section>
+        <h1><dfn>Coalesced and predicted events</dfn></h1>
+
+        <div class="note">This specification does not define how user agents should coalesce or
+        predict pointer movement data. It only specifies the API for accessing this information.</div>
+
+        <section>
+            <h2><dfn>Coalesced events</dfn></h2>
+
+            <p>For performance reasons, user agents may choose not to send a <code>pointermove</code>
+            event every time the position of a pointer is updated. Instead, they will coalesce
+            (combine/merge) multiple position changes into a single, averaged move event. While
```

Comment:
Not sure 'averaged' is what is actually happening. The even which is fired contains the most recent information about the position and such. It is just that some events which weren't fired may be put to the coalesced events list.

Reply:
ah, good point. i can probably just remove the ", averaged" and be done with it (as this spec doesn't define how these coalesced/non-coalesced events are divined, only how they're accessed)